src/main/java/games/strategy/engine/framework/startup/launcher/ServerLauncher.java

```diff
@@ -402,44 +402,44 @@ private static String getConnectionLostFileName() {
     return GameDataFileUtils.addExtension(
         "connection_lost_on_" + DateTimeFormatter.ofPattern("MMM_dd_'at'_HH_mm").format(LocalDateTime.now()));
   }
-}
-
-
-class ServerReady implements IServerReady {
-  private final CountDownLatch m_latch;
-  private final int m_clients;
 
-  ServerReady(final int waitCount) {
-    m_clients = waitCount;
-    m_latch = new CountDownLatch(m_clients);
-  }
+  static class ServerReady implements IServerReady {
```

Comment:
This is the downside when working with windows as primary system ^^
I'd really appreciate it if you could  test this!